native_client/java/Makefile

```
@@ -12,8 +14,23 @@ clean: apk-clean
 apk-clean:
 	./gradlew clean
 
-apk: apk-clean bindings
-	LIBDEEPSPEECH_SO=$(LIBDEEPSPEECH_SO) ./gradlew build
+libs-clean:
+	rm -fr libdeepspeech/libs/*/libdeepspeech.so
+
+libdeepspeech/libs/arm64-v8a/libdeepspeech.so:
+	mkdir libdeepspeech/libs/arm64-v8a/ || true
+	cp ${LIBDEEPSPEECH_SO_ARM64} libdeepspeech/libs/arm64-v8a/
+
+libdeepspeech/libs/armeabi-v7a/libdeepspeech.so:
+	mkdir libdeepspeech/libs/armeabi-v7a/ || true
+	cp ${LIBDEEPSPEECH_SO_ARMV7} libdeepspeech/libs/armeabi-v7a
+
+libdeepspeech/libs/x86_64/libdeepspeech.so:
+	mkdir libdeepspeech/libs/x86_64/ || true
+	cp ${LIBDEEPSPEECH_SO_ARMV7} libdeepspeech/libs/x86_64//
```

Remove the extra/trailing slash:
```suggestion
	cp ${LIBDEEPSPEECH_SO_ARMV7} libdeepspeech/libs/x86_64/
```